webviz_subsurface/ert_jobs/smry2arrow.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+"""Convert UNSMRY file to Apache Arrow IPC file format (also known as Feather V2)
+"""
+
+from typing import List, Iterable, Dict
+import argparse
+import re
+from pathlib import Path
+import logging
+import json
+import time
+
+from ecl.summary import EclSum, EclSumKeyWordVector
+import pyarrow as pa
+import pyarrow.feather as feather
+
+logger = logging.getLogger(__name__)
+
+
+DESCRIPTION: str = """
+Convert UNSMRY file to Apache Arrow IPC file format (also known as Feather V2)
+"""
+CATEGORY: str = "utility.eclipse"
+EXAMPLES: str = """
+Convert and output summary data in SOME.UNSMRY file to an Arrow IPC file named unsmry.arrow
+this in the ert workflow:
+
+    FORWARD_MODEL SMRY2ARROW(<INPUT>=eclipse/model/SOME.UNSMRY, <OUTPUT>=share/results/tables/unsmry.arrow)
+
+"""  # noqa
```

Comment:
```suggestion
"""
```
I think `noqa` is `flake8` only (and we use `pylint`). @lindjoha remove it in your job as well?